src/components/Login.js

```diff
@@ -104,75 +105,76 @@ class Login extends Component {
-                                onChange={this.onChange}
-                                label='Username'
-                                required
-                                placeholder='Enter your username...' />
-                            <Form.Input
-                                type={showPassword ? 'text': 'password'}
-                                icon={<Icon name={showPassword ? 'eye slash outline': 'eye'} onClick={this.handleShow} link />}
-                                name="password"
-                                value={this.state.password}
-                                onChange={this.onChange}
-                                label='Password'
-                                required
-                                placeholder='Enter your password...' />
-                            <Button
-                                fluid
-                                primary
-                                type='button'
-                                onClick={this.submitLogin}
-                                disabled={!this.state.username || !this.state.password}
-                                >LOG IN</Button>
-                        </Form>
+                <Grid>
+                    <Grid.Row>
+                        <Grid.Column width={5}>
+                            <div className='login'>
+                                <span><b>Login</b></span>
+                                <Divider />
+                                <Form>
+                                    <Form.Input
+                                        name="username"
+                                        value={this.state.username}
+                                        onChange={this.onChange}
+                                        label='Username'
+                                        required
+                                        placeholder='Enter your username...' />
+                                    <Form.Input
+                                        type={showPassword ? 'text' : 'password'}
+                                        icon={<Icon name={showPassword ? 'eye slash outline' : 'eye'} onClick={this.handleShow} link />}
+                                        name="password"
+                                        value={this.state.password}
+                                        onChange={this.onChange}
+                                        label='Password'
+                                        required
+                                        placeholder='Enter your password...' />
+                                    <Button
+                                        fluid
+                                        primary
+                                        type='button'
+                                        onClick={this.submitLogin}
+                                        disabled={!this.state.username || !this.state.password}
+                                    >LOG IN</Button>
```

Comment:
@codesankalp i think we should add a preloader for better transition.

Reply:
Actually, this issue is related to adding GitHub Login. In Login.js I added a single line for including the GitHubAuth.js component and other changes are just reformatting of code. 
I think creating a new issue for this in the future will be good instead of fixing in this PR.
@nlok5923